lib/jscrewit.js

```diff
@@ -5971,7 +5973,6 @@ var wrapWithEval;
                 }
                 else
                 {
-                    var return_String = this.findDefinition(OPTIMAL_RETURN_STRING);
```

Comment:
Cuz I thought I broke the system, I redownload and made changes, so there may be some resorting differences